ops/opset13/opset13_test.go

```diff
@@ -133,6 +133,26 @@ func TestGetOperator(t *testing.T) {
 			newGRU(),
 			nil,
 		},
+		{
+			"Less",
+			newLess(),
+			nil,
+		},
+		{
+			"LessOrEqual",
+			newLessOrEqual(),
+			nil,
+		},
+		{
+			"LinearRegressor",
+			newLinearRegressor(),
+			nil,
+		},
+		{
+			"LogSoftmax",
+			newLogSoftmax(),
+			nil,
+		},
 		{
```

Comment:
Why these changes?

Reply:
These operators were not added to the tests in earlier pull request.

Reply:
Maybe nice to create an issue for it and mention it in this pull request so it is clear in the hit history that this also changed?